### PR TITLE
Feature/base database structure

### PR DESCRIPTION
**Pull request by przwojtkow**
Just for test result validation, persistence.xml file used as a main configuration file. This will be changed in the next PR.

### CODE REVIEW

File: mrchecker-framework-modules/mrchecker-database-module/src/main/java/com/capgemini/mrchecker/database/core/BaseDatabase.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import com.capgemini.mrchecker.test.core.analytics.IAnalytics;
+import com.capgemini.mrchecker.test.core.base.environment.IEnvironmentService;
+import com.capgemini.mrchecker.test.core.base.properties.PropertiesSettingsModule;
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.google.inject.Guice;
+import lombok.Getter;
+
+import javax.persistence.EntityManager;
+
+abstract public class BaseDatabase implements ITestObserver, IDatabasePrefixHolder {
```
Comment (lstefaniszyn) [inline mrchecker-framework-modules/mrchecker-database-module/src/main/java/com/capgemini/mrchecker/database/core/BaseDatabase.java]: Please we keep naming convention 
instead "BaseDatabase", could be "BasePageDatabase" ?

**Comment by lstefaniszyn** [comment]
@przwojtkow: CR done.   Please include Jenkins Job outcome.

**Comment by przwojtkow** [comment]
Jenkins job output: http://10.40.234.40:8080/job/MrChecker-SDLC/job/CI_mrchecker-database-module/52/console

**Comment by lstefaniszyn** [comment]
@przwojtkow : please move changed files into new repo: https://github.com/devonfw-forge/mrchecker-source and remove this branch.

**Comment by przwojtkow** [comment]
PR moved to the new repo.

**Pull request closed without merging**